src/main/resources/format-pages.txt

```diff
@@ -336,6 +336,21 @@ privateSpecification = true
 reader = SDTReader
 mif = true
 
+[Big Data Viewer]
+extensions = .xml, .h5
+owner = `Tobias Pietzsch <https://imagej.net/BigDataViewer/>`_
```

Comment:
Looks like https://imagej.net/BigDataViewer/ is leading us nowhere and https://imagej.net/BigDataViewer should be used instead